src/components/ChatEntry.js

```diff
@@ -1,22 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  // const onLike = () => {
+  //   props.updateLike(props.id);
+  // }
+
+  const [isLiked, setIsLiked] = useState(false);
+
+  const likeFill = isLiked ? '❤️' : '🤍';
+
+  const toggleLiked = () => {
+    setIsLiked(!isLiked);
+  };
+
+  const onLike = () => {
+    props.updateLike(props.id);
+    toggleLiked();
+  };
+
+  // This line of code is defining a function called onLike. When this function
+  //is called, it will call the updateLike function, passing in the props.id as an argument.
+  // The updateLike function is likely a function that updates the like status of the item with the given id.
+  // The onLike function will be used later in the code as an event handler for a button or some other UI element
+  //that allows the user to like or unlike an item.
+
   return (
     <div className="chat-entry local">
```

Comment:
I see you didn't do the optional enhancement, just in case you were wondering about how it worked you could basically use the ids of each message to decide if it should be on the right or left side of the screen. the receiver would always be an even id and the sender would always be odd. 

`const localRemote =
    props.id  % 2  === 0 ? 'chat-entry local' : 'chat-entry remote';
`

then  we could assign could put this class name in the `div` on line 31